clang/include/clang/Sema/Sema.h

```diff
@@ -12836,6 +12834,31 @@ class Sema final {
   }
 };
 
+template <typename AttrType>
+void Sema::addIntelSYCLSingleArgFunctionAttr(Decl *D,
+                                             const AttributeCommonInfo &CI,
+                                             Expr *E) {
+  assert(!E);
```

Comment:
Assert is the wrong way, also needs a message.

Reply:
Sorry about this. Thanks Erich. Fixed now.